prdoc/pr_3065.prdoc

```diff
@@ -0,0 +1,14 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
```

Comment:
I don't think we need a prdoc for bumping the version.

Reply:
yeah, but is there a way to make the CI pass without it?

Reply:
Use `ro-silent` label